proxy/src/state.rs

```diff
@@ -10,6 +10,10 @@ pub struct ProxyConfig {
     /// main entrypoint for users to connect to
     pub proxy_address: SocketAddr,
 
+    /// internally used for status and prometheus metrics
+    pub http_address: SocketAddr,
+
+    // TODO it's not http? it's postgres protocol over tcp.
```

Comment:
It says it's http but looks like postgres over tcp. Did I misunderstand the code or should I fix the docs? Why would we use a pg backend for auth callbacks from cplane?

Reply:
Comment is wrong. That pg endpoint was a quick hack and coded before we had an http endpoints in our rust code. Deserves to be migrated to http. Let's update the comment for now (and may be create an issue to migrate it to http).